website/source/docs/agent/configuration/server.html.md

```diff
@@ -131,24 +131,26 @@ server {
   cluster again when starting. This flag allows the previous state to be used to
   rejoin the cluster.
 
+- `server_join` - Specifies the [Server Join][server_join] stanza for
```

Comment:
Use the same syntax as the rest of the parameters. `<Name of the parameter> <type> - <Text describing>`